src/utils/text_display.jl

```diff
@@ -28,4 +28,74 @@ function human_readable_name(name::String; title_case::Bool=false)::String
     end
 
     return replace(name, "_" => " ")
-end
\ No newline at end of file
+end
+
+"""
+    to_scientific(x::Float64; kwargs...)::String
+    to_scientific(x::Int64; kwargs...)::String
+
+Converts any Real number to scientific notation.
+
+# Arguments
+- `x` : Number to be converted
+- `digits` : Optional argument. If present, represent the number of decimal digits. Defaults
+to 2.
+"""
+function to_scientific(x::Float64; digits=2)::String
+    x == 0.0 && return "0.0"
+    a, b = get_scientific_factors(x::Float64; digits=digits)
+    return "$(a)E$(b)"
+end
+function to_scientific(x::Int64; digits=2)::String
+    return to_scientific(Float64(x); digits=digits)
+end
+
+"""
+    get_scientific_factors(x::Float64; digits=2)::Tuple
+    get_scientific_factors(x::Int64; digits=2)::Tuple
+
+Return a and b to write any Real number x as x = a * 10^b, where 1.0 <= a < 10.0 and b is
+an integer. The idea is that `x = a * 10^b` => `log(x) = log(a) + b*log(10)`. Which means
+that `log(a)` is the remainder of `log(x)/log(10)` hence `a = exp((log(x)%log(10)))`.
```

Comment:
Maybe use $a\approx exp((log(x)%log(10)))$. I know it's basically equality but makes it clear.

Reply:
Actually I think you're talking about compuational approximation not mathematical sorry, so ignore me